test/sandbox/routes/v3/search/order.js

```diff
@@ -1,9 +1,19 @@
 var emptyOrder = require('../../../fixtures/v3/search/emptyOrder.json')
 var order = require('../../../fixtures/v3/search/order.json')
+var filteredOrders = require('../../../fixtures/v3/search/filter/order-filter')
+
+var NORTH_WEST_REGION = '824cd12a-6095-e211-a939-e4115bead28a'
 
 exports.order = function(req, res) {
   if (req.body.company === '375094ac-f79a-43e5-9c88-059a7caa17f0') {
     return res.json(order)
   }
+  if (
```

Comment:
Why not to use a simple condition?
```suggestion
  if (req.body.uk_region === '824cd12a-6095-e211-a939-e4115bead28a') {
```

Reply:
The simple condition doesn't work if multiple regions are selected, which the test doesn't do anyway, but it could be quite confusing for anyone who is playing with it while it's connected to the sandbox, to see the results disappear when you add another region to `Nort West`.

Reply:
So I prefer to keep it as it is.

Reply:
Good point, perhaps it's worth adding a comment.